docs/detections/rules-ui-manage.asciidoc

```diff
@@ -109,9 +109,7 @@ NOTE: When duplicating a rule with exceptions, you can choose to duplicate the r
 [[manually-run-rules]]
 === Run rules manually
 
-beta::[]
-
-Manually run enabled rules for a specified period of time for testing purposes or additional rule coverage. 
+Manually run enabled rules for a specified period of time to force test them, provide additional rule coverage, or fill gaps in rule executions.
```

Comment:
I think force sounds a bit odd here

Reply:
What do you think about using "deliberately" or "intentionally"? For example:

```suggestion
Manually run enabled rules for a specified period of time to deliberately test them, provide additional rule coverage, or fill gaps in rule executions.
```

Reply:
Thanks, this is better!